src/lib/rnp.cpp

```diff
@@ -5433,11 +5429,7 @@ try {
     }
     ret = RNP_SUCCESS;
 done:
-    if (op->password) {
-        pgp_forget(op->password, strlen(op->password) + 1);
-        free(op->password);
-        op->password = NULL;
-    }
+    op->password.resize(0);
```

Comment:
Seems odd instead of `op->password.clear()` (should be same effect on modern STLs I believe)